curriculum/challenges/english/25-front-end-development/lab-blog-post-card/66eaddd04a9e533fba689001.md

```diff
@@ -241,9 +241,16 @@ assert.exists(postImg);
 
 const imgWidth = getComputedStyle(postImg).width;
 const cardWidth = getComputedStyle(postImg.parentElement).width;
-assert.strictEqual(imgWidth, cardWidth);
+assert.closeTo(imgWidth, cardWidth, 1.5); //I decided to use 1.5 as the delta value, but IDK what it should really be
```

Comment:
test with the code from the forum if it passes: https://forum.freecodecamp.org/t/design-a-blog-post-card-design-a-blog-post-card/741125, if it does it should be fine, if there are situations in which it needs a bigger delta the test can be updated again